src/wp-includes/option.php

```diff
@@ -1106,6 +1113,71 @@ function delete_option( $option ) {
 	return false;
 }
 
+/**
+ * Determine the appropriate autoload value based on input.
+ *
+ * This function checks the provided autoload value and returns a standardized
+ * value ('yes', 'no', 'default-yes', or 'default-no') based on specific conditions.
+ *
+ * @since 6.4.0
+ *
+ * @param mixed $autoload The autoload value to evaluate.
+ *
+ * @return string The standardized autoload value: 'yes', 'no', 'default-yes', or 'default-no'.
```

Comment:
There shouldn't be a blank line between `@param` and `@return`.
```suggestion
 * @param mixed $autoload The autoload value to evaluate.
 * @return string The standardized autoload value: 'yes', 'no', 'default-yes', or 'default-no'.
```